code-challenges.test.js

```diff
@@ -45,23 +103,105 @@ const studyMinutesWeek2 = {
   wednesday: 60,
   thursday: 20,
   friday: 15,
-  saturday: 65
-}
+  saturday: 65,
+};
 // Expected output: [10, 15, 20, 45, 60, 65, 100]
 
+const studyMinutesWeek3 = {
+  sunday: 100,
+  monday: 10,
+  tuesday: 200,
+  wednesday: 260,
+  thursday: -30,
+  friday: -122,
+  saturday: 65,
+};
+
+// Expected output: [-122, -30, 10, 65, 100, 200, 260]
+
+describe("ascendingArray", () => {
+  it("The function that takes in an object and returns an array of the numbers sorted from least to greatest", () => {
+    expect(ascendingArray(studyMinutesWeek1)).toEqual([
+      15, 15, 20, 30, 30, 60, 90,
+    ]);
+    expect(ascendingArray(studyMinutesWeek2)).toEqual([
+      10, 15, 20, 45, 60, 65, 100,
+    ]);
+    expect(ascendingArray(studyMinutesWeek3)).toEqual([
+      -122, -30, 10, 65, 100, 200, 260,
+    ]);
+  });
+});
+
+//ReferenceError: ascendingArray is not defined
+
 // b) Create the function that makes the test pass.
 
+// ============== <<<<< PSEUDO CODE >>>>> ===========================
+//we need a function that will take in an object and pull the desired values (numbers)
+//since this is not an array, we  need to use Object.values function to access the values only
+//we are going to chain .forEach function so we can iterate the obj elements. Each value/element will be pushed into our newArray, one at a time.
+//the function will return sorted version of our new array
+
+const ascendingArray = (obj) => {
+  const newArray = [];
+  Object.values(obj).forEach((val) => {
+    newArray.push(val);
+  });
+  return newArray.sort((a, b) => a - b);
```

Comment:
You can chain .sort to the array given from Object.values removing the need for .forEach.